code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -643,11 +764,14 @@
 		qdel(O)
 	for (var/mob/M in bay.GetAllContents())
 		qdel(M)
+	for (var/turf/T in bay)
+		T.ChangeTurf(/turf/open/floor/plasteel)
 
 /datum/centcom_podlauncher/Destroy() //The Destroy() proc. This is called by ui_close proc, or whenever the user leaves the game
-	updateCursor(FALSE, FALSE) //Make sure our moues cursor resets to default. False means we are not in launch mode
+	updateCursor(TRUE) //Make sure our moues cursor resets to default. False means we are not in launch mode
 	qdel(temp_pod) //Delete the temp_pod
 	qdel(selector) //Delete the selector effect
+	qdel(indicator)
```

Comment:
null it as well as deleting, to prevent a hard delete

Reply:
ie QDEL_NULL?